workshop/exercise-2/README.md

```diff
@@ -14,21 +14,52 @@ The application for this lab is a simple node.js with express app which returns
 	cd fib-knative
 	```
 
+## Set up a Private Container Registry and Obtain Credentials
+In this lab, we'll use the IBM Container Registry to host our docker images since you already have access to this through your IBM Cloud Account. IBM Container Registry enables you to store and distribute container images in a fully managed private registry.
+
+1. Install the container registry plugin:
+
+  ```
+  ibmcloud plugin install container-registry
+  ```
+2. Choose a name for your first namespace, and create it. 
```

Comment:
Perhaps add something like: `A namespace represents your private spot within the registry that holds all of your images that only you have access to.` - just to explain what it is and why we're creating it.